sdk/src/wallet/account/operations/output_claiming.rs

```diff
@@ -242,6 +242,15 @@ where
         let mut available_amount = 0;
         let mut required_amount_for_nfts = 0;
         let mut new_native_tokens = NativeTokensBuilder::new();
+
+        // There can be outputs with less amount than min required storage deposit, so we have to check that we
+        // have enough amount to create a new basic output
+        let enough_amount_for_basic_output = possible_additional_inputs
+            .iter()
+            .map(|i| i.output.amount())
```

Comment:
What if there are returns in the possible_additional_inputs and in the end the actual sum is not enough ?

Reply:
I checked the code, and these additional_inputs are unlock_conditions().len() == 1, so allegedly only AddressUnlockCondition outputs. We may want to... foolproof that check :)

Reply:
It's also documented for `get_basic_outputs_for_additional_inputs()` that there will only be the address unlock condition, exactly for this reason that we don't have to take care of sdruc or check timelock expiration again